README.md

```diff
@@ -18,20 +17,16 @@ Search API provides access to this metadata.
 
 - Ingest
   - Ingest refers to the process of validating, inserting, updating, or
-    deleting metadata in the CMR system and affects only the metadata for the
-    specific Data Partner. The CMR allows Data Partners to ingest metadata
+    deleting metadata in the CMR system and affects only the metadata for the specific Data Partner. The CMR allows Data Partners to ingest metadata
     records through a RESTful API
   - API Docs: https://cmr.earthdata.nasa.gov/ingest/site/ingest_api_docs.html
 
 - Access Control
   - Access Control Lists (ACLs) are the mechanism by which users are granted
-    access to perform different operations in the CMR. CMR ACLs follow the
-    same design as ECHO ACLs which in turn are derived from the generic ACL
+    access to perform different operations in the CMR. CMR ACLs follow the same design as ECHO ACLs which in turn are derived from the generic ACL
     design pattern used in many other systems. At a high level, an ACL is a
     mapping of actors (subjects) to resources (object) to operations
-    (predicate). For instance, a CMR ACL might specify that all Registered
-    users have READ access to ASTER data or all users in a provider operations
-    group have permissions to ingest data for a particular provider.
+    (predicate). For instance, a CMR ACL might specify that all Registered users have READ access to ASTER data or all users in a provider operations group has permissions to ingest data for a particular provider.
```

Comment:
"have" was correct; the change to "has" is not grammatically correct ("all users ... have").